File: src/test/java/no/nav/syfo/testhelper/UserConstants.java
package no.nav.syfo.testhelper;

import no.nav.syfo.mock.AktoerMock;

import static no.nav.syfo.mock.BrukerprofilMock.PERSON_ETTERNAVN;
import static no.nav.syfo.mock.BrukerprofilMock.PERSON_FORNAVN;

public class UserConstants {

    public static final String ARBEIDSTAKER_FNR = "12345678912";
    public static final String ARBEIDSTAKER_AKTORID = AktoerMock.mockAktorId(ARBEIDSTAKER_FNR);
    public static final String LEDER_FNR = "12987654321";
    public static final String LEDER_AKTORID = AktoerMock.mockAktorId(LEDER_FNR);
    public static final String VIRKSOMHETSNUMMER = "123456789";

    public final static String PERSON_NAVN = PERSON_FORNAVN + " " + PERSON_ETTERNAVN;
}

File: src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt
package no.nav.syfo.testhelper

import no.nav.syfo.consumer.aktorregister.AktoerMock.Companion.mockAktorId

object UserConstants {
    const val ARBEIDSTAKER_FNR = "12345678912"
    val ARBEIDSTAKER_AKTORID = mockAktorId(ARBEIDSTAKER_FNR)
    const val LEDER_FNR = "12987654321"
    val LEDER_AKTORID = mockAktorId(LEDER_FNR)
    const val VIRKSOMHETSNUMMER = "123456789"
}
